src/spyglass/decoding/v1/clusterless.py

```diff
@@ -279,15 +279,19 @@ def make(self, key):
 
         nwb_file_name = key["nwb_file_name"].replace("_.nwb", "")
 
+        # Make sure the results directory exists
+        results_dir = Path(config["SPYGLASS_ANALYSIS_DIR"]) / nwb_file_name
+        results_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
Is this a case you hit? I thought `settings.py` did this when reading the config here: https://github.com/LorenFrankLab/spyglass/blob/4a198e5f55744388c930e05e281d36c68c1c84c8/src/spyglass/settings.py#L255-L259